src/msg/Dispatcher.h

```diff
@@ -94,8 +94,8 @@ class Dispatcher {
    * other threads, so any locks it takes must be at the very bottom of the hierarchy).
    * Messages are delivered in receipt order within a single Connection, but there are
    * no guarantees across Connections. This makes it useful for some limited
-   * coordination between Messages which can be fast_dispatch'ed and those which must
-   * go through normal dispatch.
+   * coordination between Messages which can go through fast_dispatch and those which
+   * must go through normal dispatch.
```

Comment:
Was this really done by code?  I'm impressed.

Reply:
No. This was me reviewing after the fact and realizing my default correction was bad. I did look around and discovered that what I wrote here coincidentally matched style.